docs/development/frameworks/apache-tomcat-on-centos-8/index.md

```diff
@@ -0,0 +1,140 @@
+
+    {{< content "limited-user-note-shortguide" >}}
+
+1. Install and set Java Home Environment [Java Development Kit](/docs/development/java/). Run the following commands to check the version of java that is installed.
+
+        java -version
+        javac -version
+
+1.  Install the `wget` and `tar` utilities. You will need this in a later section to install the Apache Tomcat 9.
+
+        sudo yum install wget -y && sudo yum install tar
+
+## Download and Install Apache Tomcat
+
+1. Create a directory to download Apache Tomcat 9:
+
+        sudo mkdir /usr/local/tomcat
+
+1. Change to `/usr/local/tomcat` and download Apache Tomcat 9. As of writing this guide, Tomcat 9.0.33 is the latest version. See [Apache Tomcat's download page](https://tomcat.apache.org/download-90.cgi) for their latest core tarball:
+
+        sudo wget https://archive.apache.org/dist/tomcat/tomcat-9/v9.0.33/bin/apache-tomcat-9.0.33.tar.gz
+
+      {{< caution >}}
+Ensure that the version number matches the Tomcat 9 version you wish to download.
+{{< /caution >}}
+
+1. Extract the downloaded tarball's contents into `/usr/local/tomcat` directory:
+
+        sudo tar xvf apache-tomcat-9.0.33.tar.gz --strip-components=1 -C /usr/local/tomcat
+
+1. Create a symbolic link to the latest version of Tomcat, that points to the Tomcat installation directory:
+
+        sudo ln -s /usr/local/tomcat/apache-tomcat-9.0.33 /usr/local/tomcat/tomcat
+
+1. Create a `tomcat` user and change the directory ownership to `tomcat`:
+
+        sudo useradd -r tomcat
+        sudo chown -R tomcat:tomcat /usr/local/tomcat
+
+1.  Create a new `systemd` service file, `/etc/systemd/system/tomcat.service`, in the text editor of your choice with the following details:
+
+      {{< file "/etc/systemd/system/tomcat.service" service >}}
+[Unit]
+Description=Tomcat Server
+After=syslog.target network.target
+
+[Service]
+Type=forking
+User=tomcat
+Group=tomcat
+
+Environment=JAVA_HOME=/usr/lib/jvm/jre        Environment='JAVA_OPTS=-Djava.awt.headless=true'
+Environment=CATALINA_HOME=/usr/local/tomcat
+Environment=CATALINA_BASE=/usr/local/tomcat
+Environment=CATALINA_PID=/usr/local/tomcat/temp/tomcat.pid
+Environment='CATALINA_OPTS=-Xms512M -Xmx1024M'
+ExecStart=/usr/local/tomcat/bin/catalina.sh start
+ExecStop=/usr/local/tomcat/bin/catalina.sh stop
+
+[Install]
+WantedBy=multi-user.target
+{{< /file >}}
+
+1. Reload the `systemd` daemon to let it know about the `tomcat.service` that you created:
+
+        sudo systemctl daemon-reload
+
+1. Start and enable Tomcat server:
+
+        sudo systemctl enable tomcat
+        sudo systemctl start tomcat
```

Comment:
I reversed the order of these - enable first, then start. I tried to start first, and it didn't know what it was that I was starting because it didn't have the systemd sym links setup yet.